### PR TITLE
Put the network UI and overlay docs in separate files

### PR DESCRIPTION
**Pull request by squaremo**
.. and link to them separately. They are different features. Content is unchanged.

Signed-off-by: Michael Bridgen mikeb@squaremobius.net

### CODE REVIEW

File: experimental/overlay.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+## Native Multi-host networking
```
Comment (moxiegirl) [inline experimental/overlay.md]: One hash for the H1

**Comment by moxiegirl** [comment]
@icecrime Totally up to you if you want to split this info.  Makes sense if people are likely to edit these individually.

LGTM

**Comment by icecrime** [comment]
Feels a bit weird as the networking doc mentions:

>    - `network` and `service` objects are globally significant and provides multi-host container connectivity natively
> - Inbuilt simple Service Discovery
>   - With multi-host networking and top-level `service` object, Docker now provides out of the box simple Service Discovery for containers running in a network

@mavenugo @mrjana Thoughts?

**Comment by squaremo** [comment]
@icecrime My motivation was that "Multi-host networking" linked to the network UI, and only when you scrolled down quite far did you see anything about the overlay driver.

Of course, if the network and services UI is tied to the overlay network (is this what you are suggesting?), then the better solution is to change the text of the link. I don't know where this would leave other drivers, and network plugins, though.

**Comment by mrjana** [comment]
@squaremo Can you please rebase and it should be ready for merge? Also can you please change the file from `overlay.md` to `networking-overlay.md` so that it's qualified properly since in docker/docker overlay can mean other things?

**Comment by squaremo** [comment]
> since in docker/docker overlay can mean other things

Good point; I've renamed `overlay.md` -> `network_overlay.md` (seemed to fit local customs better). And rebased on master.

**Comment by jfrazelle** [comment]
LGTM

**Comment by squaremo** [comment]
Cheers all!

**Pull request merged**